sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/CommonTestFramework/CustomXunitAttributes.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Xunit
+{
+    public class CustomXunitAttributes
+    {
+        [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+        public class ConditionallySkipOSFactAttribute : FactAttribute
+        {
+            private readonly string _platformToSkip;
+            private readonly string _reason;
+
+            public ConditionallySkipOSFactAttribute(string platformToSkip, string reason)
```

Comment:
How about using OSPlatform type instead of string platformToSkip?

Reply:
That was my original intent. :) 
OSPlatform isn't a const so it can't be used in an attribute.